ocs_ci/ocs/constants.py

```diff
@@ -305,3 +308,21 @@
 # Default SC based on platforms
 DEFAULT_SC_AWS = "gp2"
 DEFAULT_SC_VSPHERE = "thin"
+
+# ignition files
+BOOTSTRAP_IGN = "bootstrap.ign"
+MASTER_IGN = "master.ign"
+WORKER_IGN = "worker.ign"
+
+# vSphere related constants
+VSPHERE_INSTALLER_REPO = "https://github.com/openshift/installer.git"
+VSPHERE_DIR = os.path.join(EXTERNAL_DIR, "installer/upi/vsphere/")
+INSTALLER_IGNITION = os.path.join(VSPHERE_DIR, "machine/ignition.tf")
```

Comment:
Do you think its a good idea to prfix VMWARE for the constants here, like VMWARE_INSTALLER_ROUTE53. Because in future if multiple platforms want the ROUTE53 constant then it could clash with this name. Also prefixing will increase more readability that this constant is more specific to VMWARE platform

Reply:
Agreed. I will handle it in followup PR